### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by Juanse7793**
In this project, I choose the Covid-19 API to create a SPA showing the Covid cases in South America.

## Requirements
- First I choose the [Narrativa API](https://covid19tracking.narrativa.com/index_en.html): COVID-19 data.

### Home page
- When the page loads, the web app shows the list of countries filtered to show only the south American countries.
- When the user clicks (or taps) on a country item, the application navigates to the details page.

### Details page
- On the details page, the web app retrieves data from the API to show detailed data in the country.
- When the user clicks on the "Return" button (<), the user navigates to the home page.

### Testing
- I create a test for rendering the homepage, details page, and the app.
- I mock the access to the API
- I mock the access to redux store

- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API should be stored in the Redux store.
- Every page (the main page and the pages for each item) should have a unique route within the SPA.
- The project is deployed on netlify.

 ### Additionals
- Includes transitions
- Has an about page
- Good desktop version

### CODE REVIEW

File: src/pages/home.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        <img className="sac-img" src={southAmerica} alt="South american countries" />
+        <div className="first-card">
+          <h2 className="subtitle">South America</h2>
+          <p className="description">442.5M hab</p>
+          <button type="button" className="btn-about" onClick={handleAbout}>About</button>
+        </div>
+      </div>
+      <div className="second-card-cont">
+        <h2 className="stats">STATS BY COUNTRY</h2>
+        <div className="second-card">
```
Comment (keroloussamy) [inline src/pages/home.js]: - [x] Your project is complete, just one thing left is to add a filter by the country as mentioned on the requirements `one page with a list of items that could be filtered by some parameters; in the example below, it's a list of metrics that can be filtered by the country (imagine a search field to introduce the country name like Italy, Croatia, etc.). This page should be your homepage.`

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @Juanse7793, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/home.js (hunk 1 above)
Comment (Juanse7793) [reply src/pages/home.js]: Thank you its done now

**Review by Kingstalux — APPROVED** [review]
Hi @Juanse7793, 
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966

## Highlights 👏 
✅ Good video presentation 👍 
✅ Professional README file 💯 
✅ App has the correct functionality 👏 
✅ Responsive design 🔥 

Good job implementing the required changes 👍 Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :ship: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**